tools/build_chain.sh

```diff
@@ -1385,15 +1421,26 @@ for line in ${ip_array[*]};do
     [ "$num" == "$ip" ] || [ -z "${num}" ] && num=${node_num}
     echo "Processing IP:${ip} Total:${num} Agency:${agency_array[${server_count}]} Groups:${group_array[server_count]}"
     [ -z "$(get_value ${ip//./}_count)" ] && set_value ${ip//./}_count 0
+    sdk_path="${output_dir}/${ip}/sdk"
+    if [ ! -d ${sdk_path} ];then
+        gen_cert "${output_dir}/cert/${agency_array[${server_count}]}" "${sdk_path}" "sdk" >>"${logfile}" 2>&1
+        # FIXME: delete the below unbelievable ugliest operation in future
+        cp sdk.crt node.crt
+        cp sdk.key node.key
+        # FIXME: delete the upside unbelievable ugliest operation in future
+        rm node.nodeid
+        cp ${output_dir}/cert/ca.crt ${sdk_path}/
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5331982)